qemu/tests/numa_stress.py

```diff
@@ -87,6 +87,8 @@ def run(test, params, env):
     most_used_node, memory_used = max_mem_map_node(host_numa_node, qemu_pid)
 
     for test_round in range(test_count):
+        if os.path.isfile(memory_file):
```

Comment:
The file will be created after first round.  
@yanan-fu how many numa nodes are there in your host ?  I tested with my host which have two nodes and "os.path.isfile(memory_file)"  is true in the second round.